accord-core/src/main/java/accord/messages/BeginRecovery.java

```diff
@@ -76,19 +76,13 @@ public void process(Node node, Id replyToNode, ReplyContext replyContext)
                                          .anyMatch(c -> !c.savedDeps().contains(txnId));
 
                 // committed txns with an earlier txnid and have our txnid as a dependency
-                earlierCommittedWitness = committedStartedBefore(instance, txnId, txn.keys)
-                                          .filter(c -> c.savedDeps().contains(txnId))
-                                          .collect(Dependencies::new, Dependencies::add, Dependencies::addAll);
+                earlierCommittedWitness = committedStartedBefore(instance, txnId, txn.keys);
 
                 // accepted txns with an earlier txnid that don't have our txnid as a dependency
-                earlierAcceptedNoWitness = uncommittedStartedBefore(instance, txnId, txn.keys)
-                                              .filter(c -> c.is(Accepted)
-                                                           && !c.savedDeps().contains(txnId)
```

Comment:
trying to understand this difference... we now do `command.savedDeps().contains(txnId)`... is there a test I can look at that hits this subtle difference?